cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/CxxCoverageSensor.java

```diff
@@ -247,7 +247,7 @@ private void saveZeroValueForResource(InputFile inputFile, SensorContext context
             }
           }
           measuresTotal.putAll(cacheCov.get(report.getAbsolutePath()));
```

Comment:
will throw a NullPointerException because `cacheCov.get(report.getAbsolutePath())`returns `null`

Reply:
`parseCoverageReport' XMLStreamException is not working in case of empty report?
Strange: Throwing first and then trying to write a message?

```Java
try {
       parser.processReport(context, report, measuresForReport);
     } catch (XMLStreamException e) {
       throw new EmptyReportException("Coverage report" + report + "cannot be parsed by" + parser, e); 
     }
 
     if (measuresForReport.isEmpty()) {
       LOG.warn("Coverage report {} result is empty (parsed by {})", report, parser);
       return false;
}
```